rqd/rqd.example.conf

```diff
@@ -12,6 +12,8 @@ OVERRIDE_NIMBY=True
 CHECK_INTERVAL_LOCKED = 60
 # Seconds of idle time required before nimby unlocks.
 MINIMUM_IDLE = 900
+# Url to the rqd project on sentry
```

Comment:
It might be helpful to add a brief explanation of what the Sentry DSN is for users who might not be familiar with Sentry. For example:

```
# Url to the rqd project on Sentry. This is used to send error and performance data to Sentry.
# SENTRY_DSN_PATH=http://sentry.yourdomain.com/40
```